test-framework/sudo-compliance-tests/src/sudoers/timestamp_timeout.rs

```diff
@@ -12,6 +12,17 @@ Defaults timestamp_timeout=0.1"
     .user(User(USERNAME).password(PASSWORD))
     .build()?;
 
+    // input valid credentials
+    // try to sudo without a password
+    Command::new("sh")
```

Comment:
I would put the "pass" scenario in a separate test function: "credential caching works with custom timeout"